srcalc/src/test/java/gov/va/med/srcalc/domain/model/ProcedureTest.java

```diff
@@ -30,4 +31,47 @@ public final void testEquals()
         EqualsVerifier.forClass(Procedure.class).verify();
     }
     
+    @Test(expected = IllegalArgumentException.class)
+    public final void testCptTooShort()
```

Comment:
The opposite of these methods should be tested as well if we are requiring them to be in a range. (i.e. cptTooLong, longDescTooShort).